index.js

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2015 Colin Clark
+
+Licensed under the 3-Clause "New" BSD license.
+You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the 3-Clause "New" BSD License at
+https://github.com/colinbdclark/infusion-electron/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion");
+require("./src/app.js");
+require("./src/core.js");
+require("./src/window.js");
+
```

Comment:
And in the case of a web script using <code>core.js</code>? Is it better to go with the mild duplication of doing something like this there?

<code>electron.ipc = electron.ipc || require("ipc");</code>